### DISCUSSION TITLE
manjaro-sway-settings 15.6.0: where do I find kvantum theme "Matchama-Dark"

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#629] [Q&A] [answered]
manjaro-sway-settings 15.6.0 changed kvantum-theme to `Matchama-Dark`: https://github.com/manjaro-sway/desktop-settings/commit/5f64ebabab6aa00202f98237b9efaeaaf17acd5b

This does not seem to exist on my machine. Now, every time I login the kvantum manager pops up and asks me to select a theme folder:

![swappy-20231125_144755](https://github.com/manjaro-sway/manjaro-sway/assets/274236/2023c65e-e789-47cc-acdf-ddc74098250d)

[This comment](https://github.com/manjaro-sway/manjaro-sway/issues/250#issuecomment-1042737597) seems to indicate that the Matchama theme does not exist any more, and `Matcha-Dark` should be used instead.

Was the change back to `Matchama-Dark` perhaps made by mistake?

**Comment by boredland** [comment]
Probably I am mistaken. Will chick in a few..

**Reply by boredland** [reply, marked as answer]
Seems somehow my install of `kvantum-theme-match` was borked, which got me all confused. will revert instantly.

**Reply by heapifyman** [reply]
Thanks. [15.6.1](https://github.com/manjaro-sway/desktop-settings/releases/tag/15.6.1) fixed it.
